Python/ceval.c

```diff
@@ -2844,6 +2854,18 @@ _PyEval_EvalFrameDefault(PyFrameObject *f, int throwflag)
             FAST_DISPATCH();
         }
 
+        TARGET(JUMP_FINALLY) {
+            PyObject *exc = Py_None;
+            Py_INCREF(exc);
+            Py_INCREF(exc);
+            Py_INCREF(exc);
+            PUSH(exc);
+            PUSH(exc);
+            PUSH(exc);
+            JUMPBY(oparg);
```

Comment:
Is it needed? It seems to me that JUMP_FINALLY always jumps to the next instruction.